python/lsst/pipe/tasks/postprocess.py

```diff
@@ -55,7 +56,41 @@ def flattenFilters(df, noDupCols=['coord_ra', 'coord_dec'], camelCase=False):
     return newDf
 
 
-class WriteObjectTableConfig(pexConfig.Config):
+class WriteObjectTableConnections(pipeBase.PipelineTaskConnections,
+                                  defaultTemplates={"coaddName": "deep"},
+                                  dimensions=("tract", "patch", "skymap")):
+    inputCatalogMeas = connectionTypes.Input(
+        doc="Catalog of source measurements on the deepCoadd.",
+        dimensions=("tract", "patch", "band", "skymap"),
+        storageClass="SourceCatalog",
+        name="{coaddName}Coadd_meas",
+        multiple=True
```

Comment:
I guess there's no advantage to these being `deferLoad=True` because we need to hold them all in memory at the same time anyway?

Reply:
That's right. We're vertically concatenating them.